src/ixdat/techniques/reactor.py

```diff
@@ -0,0 +1,289 @@
+        helper function to convert different units used in a TPMS measurement
+        Args:
+            original_unit (str): The original unit of a DataSeries
+            new_unit (str): The new unit for DataSerie
+        Return:
+            unit_factor, new_unit_name
+
+        """
+
+        try:
+            if original_unit == "celsius" or original_unit == "C":
+                unit_factor = {
+                    "C": 0,
+                    "celsius": 0,
+                    "K": 273.15,
+                    "kelvin": 273.15,
+                }[new_unit]
+
+            if original_unit == "1/celsius" or original_unit == "1/C":
+                unit_factor = {
+                    "1/C": 0,
+                    "1/celsius": 0,
+                    "1/K": 273.15,
+                    "1/kelvin": 273.15,
+                }[new_unit]
+
+            elif original_unit == "kelvin" or original_unit == "K":
+                unit_factor = {
+                    "K": 1,
+                    "kelvin": 1,
+                    "celsius": -273.15,
+                    "C": -273.15,
+                }[new_unit]
+
+            elif original_unit == "mbar":
+                unit_factor = {
+                    "mbar": 1,
+                    "bar": 1000,
+                    "hPa": 1,
+                    "kPa": 0.1,
+                }[new_unit]
+
+            elif original_unit == "bar":
+                unit_factor = {
+                    "mbar": 1e-3,
+                    "bar": 1,
+                    "hPa": 1e-3,
+                    "kPa": 0.1e-3,
+                }[new_unit]
+
+            else:
+                unit_factor = {
+                    # Time conversion
+                    "s": 1,
+                    "min": 1 / 60,
+                    "minutes": 1 / 60,
+                    "h": 1 / 3600,
+                    "hr": 1 / 3600,
+                    "hour": 1 / 3600,
+                    "hours": 1 / 3600,
+                    "d": 1 / (3600 * 24),
+                    "days": 1 / (3600 * 24),
+                    # Pressure conversion
+                    "mbar": 1,
+                    "bar": 1000,
+                    "hPa": 1,
+                    "kPa": 0.1,
+                    # Temperature conversion
+                    "K": 273.15,
+                    "kelvin": 273.15,
+                }[new_unit]
+        except KeyError:
+            warnings.warn(
+                f"Can't convert original unit '{original_unit}' to new unit"
+                f"'{new_unit}'. You can try to clear cache with "
+                "measurement.clear_cache(). Using original unit {original_unit} !",
+                stacklevel=2,
+            )
+            unit_factor = 1
+            new_unit = original_unit
+        return unit_factor, new_unit
+
+
+class SpectroReactorMeasurement(ReactorMeasurement, SpectroMSMeasurement):
+    default_plotter = SpectroTPMSPlotter
+
+
+class ReactorUnitConverterResult(Saveable):
```

Comment:
So this class was needed in the idea that unit conversion should be done through calibration series since DataSeries are thought as immutable. But a method correcting data in the original dataseries has been implemented and made this obsolete. Furthermore now with pint in the picture that might have made the entire unit conversion method obsolete